projects/1.19/assets/red-power/red_power/lang/zh_cn.json

```
@@ -0,0 +1,108 @@
+{
+  "itemGroup.red_power": "Red Power",
+
+  "death.attack.wire": "%1$s被高压电烤熟了",
+
+  "feedback.red_power.success": "§a新链接了{#2}格长的§r{#1}",
+  "feedback.red_power.first_point": "§d第一点设至§r{#1}§7（{#2}）",
+  "feedback.red_power.reset": "§d选择已重置",
+  "feedback.red_power.fail": "§c线缆连接失败，请再次尝试",
+  "feedback.red_power.wrong_block": "{#1}§c无法连接线缆",
+  "feedback.red_power.connections_limit": "{#1}§c已达链接数上限§7（{#2}/{#3}）",
+  "feedback.red_power.cant_transmit": "{#1}§c无法传输信号",
+  "feedback.red_power.cant_receive": "{#1}§c无法接收信号",
+  "feedback.red_power.distance_limit": "§c超过长度上限§7（{#1}/{#2}）",
+  "feedback.red_power.identical_points": "§c请选择另一位置",
+  "feedback.red_power.wire_exist": "§c此处已有线缆，请选择另一位置",
+  "feedback.red_power.no_wires": "§c左手需持有线缆",
+  "feedback.red_power.no_enough_wires": "§c线缆不足§7（{#1}/{#2}）",
+
+  "tooltip.red_power.not_dyed": "未染色",
+  "tooltip.red_power.attenuation": "- 信号每{#1}格产生衰减。",
+  "tooltip.red_power.max_length": "- 最长{#1}格。",
+  "tooltip.red_power.max_connections": "- 最多允许{#1}条链接。",
+
+  "tooltip.red_power.common_wires": "基础线缆，其中信号衰减显著。",
+  "tooltip.red_power.amethyst_wires": "完全隐形的线缆。",
+  "tooltip.red_power.echo_wires": "强化线缆，其中信号会少量衰减。",
+  "tooltip.red_power.ender_wires": "强化线缆，其中信号衰减量中等。",
+  "tooltip.red_power.power_wires": "强效线缆，其中信号衰减极少。",
+  "tooltip.red_power.wrench": "允许连接无实体红石线缆。{#}§2- 右击以选择连接点。{#}§2- Shift右击以重置。",
+  "tooltip.red_power.creative_wrench": "允许连接无实体红石线缆。无视长度上限和链接数上限。",
+  "tooltip.red_power.cutters": "允许拆卸无实体红石线缆。{#}§2- 右击以选择线缆。{#}§2- Shift右击以拆卸。",
+  "tooltip.red_power.safety_glasses": "显示链接信息和节点扩展信息。",
+
+  "tooltip.red_power.reader_node": "读取所依附方块的红石信号。",
+  "tooltip.red_power.node": "接收并传输信号。",
+  "tooltip.red_power.recorder_node": "接收信号并将其输出至所依附的方块。",
+  "tooltip.red_power.amplifier_node": "能将输入信号放大至10。",
+  "tooltip.red_power.hidden_reader_node": "读取所依附方块的红石信号。完全隐形。",
+  "tooltip.red_power.hidden_node": "接收并传输信号。完全隐形。",
+  "tooltip.red_power.hidden_recorder_node": "接收信号并将其输出至所依附的方块。完全隐形。",
+  "tooltip.red_power.hidden_amplifier_node": "能将输入信号放大至10。完全隐形。",
+  "tooltip.red_power.core": "复合式节点，能将输入信号放大至15，并抑制闪电。",
+  "tooltip.red_power.not_gate": "反转输入信号。",
+  "tooltip.red_power.and_gate": "若所有输入信号均为高，则输出信号。",

```

```suggestion
  "tooltip.red_power.and_gate": "若所有输入信号均被激活，则输出信号。",
```
只是个人想法，这样表达更好理解，或者也可以用“为假”“为真”则更偏向数电术语

---

https://minecraft.fandom.com/zh/wiki/%E6%95%99%E7%A8%8B/%E5%9F%BA%E6%9C%AC%E9%80%BB%E8%BE%91%E9%97%A8#%E9%9D%9E%E9%97%A8

---

主要是信号时零点几的时候视为未激活
要不然直接上0和1了

---

确实有这个问题，不排除是作者写的时候搞出来的问题，而且他的信号衰减应该是每到标注的距离就会衰减1的，但是他衰减的数值却不是整数

---

那就这样吧，反正不懂数电的应该也不会去用逻辑门